__tests__/app.test.js

```diff
@@ -90,4 +90,58 @@ describe.only("GET /api/articles/:article_id", () => {
           console.log(response.body.msg, "No data for ID")
         })
     });
-});
\ No newline at end of file
+});
+
+let newVotes = { incVotes: 1 };
+
+describe("PATCH /api/articles/:article_id", () => {
+  test("Status 200", () => {
+    return request(app)
+      .patch("/api/articles/3")
+      .send(newVotes)
+      .expect(200)
+    })
+  test("Returns the updated article", () => {
+    return request(app)
+      .patch("/api/articles/3")
+      .send(newVotes)
+      .expect(200)
+      .then((response) => {
+        expect(Object.keys(response.body)).toHaveLength(7);
+        expect(response.body).toEqual({
+            article_id: 3,
+            title: "Eight pug gifs that remind me of mitch",
+            topic: "mitch",
+            author: "icellusedkars",
+            body: "some gifs",
+            created_at: '2020-11-03T09:12:00.000Z',
+            votes: 1,
+          },
+        );
+      });
+  })
+  test("Runs multiple times", () => {
+    return request(app)
+      .patch("/api/articles/3")
+      .send(newVotes)
+      .expect(200)
+      .then(() => {
+        newVotes.incVotes = -100;
+        return request(app)
+          .patch("/api/articles/3")
+          .send(newVotes)
+          .expect(200)
+          .then((response) => {
+            expect(response.body.votes).toBe(-99)
+          })
+      })
+  })
+  test("Status 404 when valid but non-existant :article_id", () => {
+    return request(app)
+      .patch("/api/articles/4321567")
+      .expect(404)
+      .then((response) => {
+        expect(response.body.msg).toBe("Invalid ID, no data found")
+      })
+  });
+})
```

Comment:
I think we might be missing a test or two here, think about errors that could come with the request body, basically anything a user can touch could cause an error :)